linera-views/src/views/mod.rs

```diff
@@ -96,6 +96,16 @@ pub trait View: Sized {
     }
 }
 
+/// A view which can have its context replaced.
+pub trait ReplaceContext<C: crate::context::Context>: View {
```

Comment:
I'm not sure I understand the reason for this parameter.  It's anyway fixed by `Target`.